sovrin_client/cli/genesisTxns.py

```diff
@@ -1,10 +1,10 @@
+from plenum.common.txn import STEWARD, TRUST_ANCHOR
 from plenum.common.types import f
 
-from sovrin_common.txn import TXN_TYPE, TARGET_NYM, ROLE, TXN_ID, NYM,\
-    STEWARD, SPONSOR
+from sovrin_common.txn import TXN_TYPE, TARGET_NYM, ROLE, TXN_ID, NYM
 
 STEWARD_SEED = b'steward seed used for signer....'
-SPONSOR_SEED = b'sponsors are people too.........'
+TRUST_ANCHOR_SEED = b'trust anchors are people too....'
```

Comment:
What are the consequences of changing the seed here? Presumably it invalidates any genesis transactions that have been done with the previous seed. A breaking change is not necessarily unacceptable, but I'm a bit uncertain about unexpected consequences.